jobs/devex/jenkins-bump-version/bump-jenkins.sh

```diff
@@ -42,15 +42,18 @@ setup_dist_git() {
 
 # download jenkins war
 prep_jenkins_war() {
-    wget https://updates.jenkins-ci.org/download/war/${VERSION}/jenkins.war
+    wget --no-verbose https://ftp.belnet.be/pub/jenkins/war-stable/${VERSION}/jenkins.war
+    wget --no-verbose https://ftp.belnet.be/pub/jenkins/war-stable/${VERSION}/jenkins.war.sha256
+    sha256sum --check jenkins.war.sha256
```

Comment:
Would also like to enable `set -e` on this script, otherwise command run failures in this function will not be noticed.